document for HTMLBaseElement.target (#31652)
mdn/content: files/en-us/web/api/htmlbaseelement/target/index.md

---
title: "HTMLBaseElement: target property"
short-title: target
slug: Web/API/HTMLBaseElement/target
page-type: web-api-instance-property
browser-compat: api.HTMLBaseElement.target
---

{{ApiRef("HTML DOM")}}

> The `target` property of the {{domxref("HTMLBaseElement")}} interface is a string that represents default target tab to show resulting output for hyperlinks and form elements. It is a value of [`target`](/en-US/docs/Web/HTML/Element/base#target) attribute of the {{HTMLElement("base")}} element.

Reviewer: ```suggestion
The `target` property of the {{domxref("HTMLBaseElement")}} interface is a string that represents the default target tab to show the resulting output for hyperlinks and form elements.

It reflects the [`target`](/en-US/docs/Web/HTML/Element/base#target) attribute of the {{HTMLElement("base")}} element.
```

## Value

> A string that represents target values `_blank`, `_self` (default), `_parent`, `_top`, `framename`.

Reviewer: ```suggestion
A string representing the target. Its value can be:
- the name of a {{HTMLElement("frame")}}, 
- One of the [keyword with specific values](/en-US/docs/Web/HTML/Element/base#target):`_blank`, `_self`, `_parent`, or `_top`.
```

Author: thank you **@teoli2003**

---
title: "HTMLBaseElement: target property"
short-title: target
slug: Web/API/HTMLBaseElement/target
page-type: web-api-instance-property
browser-compat: api.HTMLBaseElement.target
---

{{ApiRef("HTML DOM")}}

The `target` property of the {{domxref("HTMLBaseElement")}} interface is a string that represents the default target tab to show the resulting output for hyperlinks and form elements.

It reflects the [`target`](/en-US/docs/Web/HTML/Element/base#target) attribute of the {{HTMLElement("base")}} element.

## Value

A string representing the target. Its value can be:

> - The name of a {{HTMLElement("frame")}}

Reviewer: ```suggestion
- The name of a {{HTMLElement("frame")}}.
```
